src/in_memory_db.rs

```diff
@@ -93,7 +92,7 @@ impl JsNewWithBoxRef for Database {
         let db = U::new_db_with_context(
             &mut ctx,
             String::from(""),
-            DBOptions::new(),
+            DbOptions::new(false, KeyLength(0)),
```

Comment:
Yes you right, but this line used just for the Trait definition. We will not use any of these variables between lines `73-76` and `93-96`

Reply:
If there is `DataBase` and `InMemoryDB` doesn't have `key_length` option, shouldnt we have separate struct for those?

Reply:
This a trait so each struct can implement it for own, if we want to separate them we should define a new trait for that. Also trait implementation means we have separate structure with same definition.

Reply:
Since it has completely different use-case, we could split the trait as well?
or maybe it's like 
```
struct DBOption {
  readonly bool,
}
struct StateDBOption {
  DBOption
  key_length; 
}
```
and commit option is completely different thing (which is already done)

Reply:
Done.